Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(hidapi)
+
+add_library(hidapi hidapi/hidapi.h)
```

Comment:
Looks like HIDAPI automatically supports being built as a shared library: https://github.com/signal11/hidapi/issues/235

Is there something else I should be looking for?